.env.example

```diff
@@ -2,6 +2,7 @@ STATUS_TOKEN=
 EDXORG_BASE_URL=
 EDXORG_CLIENT_ID=
 EDXORG_CLIENT_SECRET=
+ELASTICSEARCH_INDEX=
```

Comment:
Thanks for adding this but want to know any guidelines about how to populate these fields with relevant values ?